### PR TITLE
feat: update protoc-gen-triple

### PR DESCRIPTION
**Pull request by FinalT**
1.rename protoc-gen-triple to protoc-gen-go-triple
2.support generating old triple stub code
3.tag a higher version
related issues #2479

### CODE REVIEW

File: protocol/triple/triple-tool/protoc-gen-go-triple/main.go [hunk 1]
```diff
@@ -51,19 +53,41 @@ func main() {
 		os.Exit(1)
 	}
 
-	protogen.Options{}.Run(
+	var flags flag.FlagSet
+	useOld := flags.Bool("useOldVersion", false, "print the version and exit")
```
Comment (DMwangnima) [inline protocol/triple/triple-tool/protoc-gen-go-triple/main.go:57]: @chickenlj We need to talk about this flag to make it more intuitive. And we use ```go-triple_opt=useOldVersion=true``` as temporary placeholder.

Comment (chickenlj) [reply protocol/triple/triple-tool/protoc-gen-go-triple/main.go:57]: `go-triple_opt=useOldVersion=true`

This style looks good to me as it's the same as `go-grpc_opt=xxx`

File: protocol/triple/triple-tool/protoc-gen-go-triple/main.go [hunk 2]
```diff
@@ -51,19 +53,41 @@ func main() {
...
-			for _, file := range plugin.Files {
-				if file.Generate {
-					tripleGo, err := generator.ProcessProtoFile(file.Proto)
-					if err != nil {
-						return err
-					}
-					return generator.GenTripleFile(tripleGo)
-				}
+			if *useOld {
+				return genOldTriple(plugin)
```
Comment (chickenlj) [inline protocol/triple/triple-tool/protoc-gen-go-triple/main.go:66]: Will the generated `oldTriple` sources be put in the same place and given the same package name as in earlier versions?

Comment (FinalT) [reply protocol/triple/triple-tool/protoc-gen-go-triple/main.go:66]: Keep the same as before

**Review by chickenlj — APPROVED** [review]
LGTM.

**Pull request merged**